core/src/main/resources/defaults/clustertemplates/7.2.9/gcp/flow-management.json

```diff
@@ -53,9 +53,9 @@
         "type": "CORE",
         "recoveryMode": "MANUAL",
         "template": {
-          "instanceType": "e2-standard-8",
+          "instanceType": "e2-standard-4",
           "rootVolume": {
-            "size": 150
+            "size": 50
```

Comment:
why it was modified?

Reply:
@doktoric according to discussion in jira, starting from here https://jira.cloudera.com/browse/CFM-1834?focusedCommentId=4378566&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4378566

Reply:
@vbrodetskyi I think this is not a good idea because we would like to support upgrades and the root disk size in this case is very small cc @keyki

Reply:
@doktoric got it, thank you for information, what is the minimal size we can set 100GB or 150GB?

Reply:
We went with 50 GB as the default for a long time and recently we increased it to 100 to all data hub types. Upgrade requires lots of space since we need to download multiple parcels that are large. For a period of time, it also copied to multiple locations on the root disk so we need double the parcel size space. Setting it to 100 would align with other data hubs.

Reply:
@keyki thank you for clarification.